app/src/main/java/com/goayo/debtify/model/EvenSplitStrategy.java

```diff
@@ -0,0 +1,35 @@
+package com.goayo.debtify.model;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * @author Alex Phu, Yenan Wang
+ * @date 2020-10-09
+ * <p>
+ * Strategy where the owedTotal is split evenly among the borrowers.
+ */
+public class EvenSplitStrategy implements IDebtSplitStrategy {
+
+    /**
+     * owedTotal is split evenly among the borrowers.
+     *
+     * @param borrowers The selected borrowers.
+     * @param owedTotal Amount that the borrowers are owed.
+     * @return A map with users and their respective owedTotal.
+     */
+    @Override
+    public Map<User, BigDecimal> splitDebt(Set<User> borrowers, BigDecimal owedTotal) {
+        // Splits the owedTotal by the number of borrowers
+        BigDecimal splitAmount = owedTotal
+                .divide(BigDecimal.valueOf(borrowers.size()), 10, RoundingMode.HALF_UP);
```

Comment:
Will this round up the second decimal or the first decimal or the whole number?
i.e. will: 
1.53 = 1.55? 
or 
1.53 = 1.50? 
or
1.53 = 2.0?